.github/workflows/pr.yml

```diff
@@ -162,8 +162,21 @@ jobs:
         echo "Base SHA: $NX_BASE"
         echo "Head SHA: $NX_HEAD"
 
-    - name: Run build
-      run: npx nx affected -t build --projects=svg-icons
```

Comment:
`affected` has no `--projects` , this was passed down to actuall commands which failed the runs. (there is no `eslint src --projects=svg-icons` etc )